core/src/main/java/org/openstack4j/model/compute/Keypair.java

```diff
@@ -35,5 +37,41 @@
 	 * @return the server fingerprint
 	 */
 	String getFingerprint();
+	
+	/**
+	 * 
+	 * @return the user_id for a keypair.
+	 */
+	String getUserId();
+	
+	/**
+	 * 
+	 * @return is deleted
+	 */
+	boolean getDeleted();
+	
+	/**
+	 * time created
+	 * @return
+	 */
+	Date getCreatedAt();
+	
+	/**
+	 * time updated
+	 * @return
+	 */
+	Date getUpdatedAt();
+	
+	/**
+	 * time deleted
+	 * @return
+	 */
+	Date getDeletedAt();
+	
+	/**
+	 * 
+	 * @return id of the keypair
+	 */
+	Integer getId();
```

Comment:
consider the below response content that I get from the url you given
```
{
    "keypair": {
        "fingerprint": "44:fe:29:6e:23:14:b9:53:5b:65:82:58:1c:fe:5a:c3",
        "name": "keypair-6638abdb-c4e8-407c-ba88-c8dd7cc3c4f1",
        "public_key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1HTrHCbb9NawNLSV8N6tSa8i637+EC2dA+lsdHHfQlT54t+N0nHhJPlKWDLhc579j87vp6RDFriFJ/smsTnDnf64O12z0kBaJpJPH2zXrBkZFK6q2rmxydURzX/z0yLSCP77SFJ0fdXWH2hMsAusflGyryHGX20n+mZK6mDrxVzGxEz228dwQ5G7Az5OoZDWygH2pqPvKjkifRw0jwUKf3BbkP0QvANACOk26cv16mNFpFJfI1N3OC5lUsZQtKGR01ptJoWijYKccqhkAKuo902tg/qup58J5kflNm7I61sy1mJon6SGqNUSfoQagqtBH6vd/tU1jnlwZ03uUroAL Generated-by-Nova\n",
        "user_id": "fake",
        "deleted": false,
        "created_at": "2014-05-07T12:06:13.681238",
        "updated_at": null,
        "deleted_at": null,
        "id": 1
    }
}
```
I think the id means the identity of the keypair